preproc/LED_charge_summary.ipynb

```diff
@@ -0,0 +1,1043 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2047d4be",
+   "metadata": {},
+   "source": [
+    "# TOFPET charge with LEDs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "45ac3feb",
+   "metadata": {},
+   "source": [
+    "This notebook is an example the TOFPET study when using LEDs."
```

Comment:
Typo: is an example `of a`...

Reply:
done!

Reply:
Sigue estando mal: `is an example of a the TOFPET study`: tienes que quitar el `the`.

Reply:
Y también tienes que usar el participio pasado de `switch`: `switched`.

Reply:
Done!